doc/openapi/firecrest-developers-api.yaml

```
@@ -239,20 +239,22 @@ paths:
       description: 'Create a directory at the specified path on the {X-Machine-Name} filesystem'
       tags:
         - Utilities
-      parameters:
-        - name: targetPath
-          in: query
-          description: Filesystem path
-          required: true
-          schema:
-            type: string
-          allowReserved: true
-        - name: p
-          in: query
-          description: 'No error if existing, make parent directories as needed'
-          allowEmptyValue: true
-          schema:
-            type: string
+      requestBody:
+        required: true
+        content:
+          'text/plain':
+            schema:
+              type: object
+              properties:
+                targetPath:
+                  type: string
+                p:
+                  type: string
```

```suggestion
                targetPath:
                  type: string
                  description: Filesystem path
                p:
                  type: string
                  description: No error if existing, make parent directories as needed
```

Could you keep the description of the parameters? I see in other requests you have kept it.